src/utils/sanitizeRow.ts

```diff
@@ -0,0 +1,45 @@
+import slugify from "slugify"
+import { Collection } from "../Entities"
+
+export const sanitizeRow = ({
+  title,
+  slug,
+  category,
+  description,
+  headerImage,
+  credit,
+  artist_ids,
+  gene_ids,
+  tag_id,
+  keyword,
+  price_guidance,
+  show_on_editorial,
+  is_featured_artist_content,
+}) => {
+  return {
+    title,
+    slug: sanitizeSlug(slug),
+    category,
+    description,
+    headerImage,
+    credit,
+    price_guidance: price_guidance ? Number(price_guidance) : null,
+    show_on_editorial: Boolean(show_on_editorial),
+    is_featured_artist_content: Boolean(is_featured_artist_content),
+    query: (artist_ids || gene_ids || tag_id || keyword) && {
+      artist_ids: artist_ids ? artist_ids.split(",").map(a => a.trim()) : [],
+      gene_ids: gene_ids ? gene_ids.split(",").map(a => a.trim()) : [],
```

Comment:
Do admins always strictly enter these as `id1,id2`, as opposed to `id1, id2` (i.e. does this need to be more lenient and split on comma + whitespace?)

Reply:
that's what the `.trim()` method is for, it removes whitespace from both sides of the string after we split on `,`.